### ISSUE TITLE
two-seater biplane

### ISSUE THREAD

**Issue by Yuan-rgb** [Luke100000/ImmersiveAircraft#66]
As a fan of your mod, I would like to ask you to consider adding a two-seater feature to the biplane. and could you please update the source code on GitHub to the Minecraft 1.20  version?
thank you <3

**Comment by Luke100000** [comment]
Yes, two seater is on my giant todo ^^
1.20 and 1.20.1 is pushed, check the branches.

**Comment by Cypress187** [comment]
Yeah, I assumed the bi-plane would be a two-seater when I made one.
